frontend/src/components/program/ScheduleEntryFilters.vue

```diff
@@ -0,0 +1,254 @@
+    <SelectFilter
+      v-if="loadingEndpoints !== true && loadingEndpoints.campCollaborations !== true"
+      v-model="value.responsible"
+      multiple
+      and-filter
+      :items="campCollaborations"
+      :display-field="campCollaborationDisplayName"
+      :label="$tc('components.program.scheduleEntryFilters.responsible')"
+    >
+      <template #item="{ item }">
+        <template v-if="item.exclusiveNone">
+          {{ item.text }}
+        </template>
+        <template v-else>
+          <TextAlignBaseline class="mr-1">
+            <UserAvatar :camp-collaboration="campCollaborations[item.value]" size="20" />
+          </TextAlignBaseline>
+          {{ item.text }}
+        </template>
+      </template>
+    </SelectFilter>
+    <v-skeleton-loader
+      v-else
+      type="button"
+      class="v-skeleton-loader--inherit-size"
+      height="32"
+      width="130"
+    />
+    <SelectFilter
+      v-if="loadingEndpoints !== true && loadingEndpoints.categories !== true"
+      v-model="value.category"
+      multiple
+      :items="categories"
+      display-field="short"
+      :label="$tc('components.program.scheduleEntryFilters.category')"
+    >
+      <template #item="{ item }">
+        <CategoryChip dense :category="categories[item.value]" class="mr-1" />
+        {{ categories[item.value].name }}
+      </template>
+    </SelectFilter>
+    <v-skeleton-loader
+      v-else
+      type="button"
+      class="v-skeleton-loader--inherit-size"
+      height="32"
+      width="100"
+    />
+    <SelectFilter
+      v-if="loadingEndpoints !== true && loadingEndpoints.progressLabels !== true"
+      v-model="value.progressLabel"
+      multiple
+      :items="progressLabels"
+      display-field="title"
+      :label="$tc('components.program.scheduleEntryFilters.progressLabel')"
+    >
+      <template #item="{ item }">
+        {{ progressLabels[item.value].title }}
+      </template>
+    </SelectFilter>
+    <v-skeleton-loader
+      v-else
+      type="button"
+      class="v-skeleton-loader--inherit-size"
+      height="32"
+      width="100"
+    />
+    <v-chip v-if="filterSet" label outlined @click="resetFilter">
+      <span>&ZeroWidthSpace;</span>
+      <v-icon left>mdi-close</v-icon>
+      {{ $tc('components.program.scheduleEntryFilters.clearFilters') }}
+    </v-chip>
+  </div>
+</template>
+
+<script>
+import UserAvatar from '@/components/user/UserAvatar.vue'
+import SelectFilter from '@/components/dashboard/SelectFilter.vue'
+import CategoryChip from '@/components/generic/CategoryChip.vue'
+import TextAlignBaseline from '@/components/layout/TextAlignBaseline.vue'
+import BooleanFilter from '@/components/dashboard/BooleanFilter.vue'
+import FilterDivider from '@/components/dashboard/FilterDivider.vue'
+import { mapGetters } from 'vuex'
+import { keyBy, sortBy } from 'lodash'
+import campCollaborationDisplayName from '@/common/helpers/campCollaborationDisplayName.js'
+
+function filterEquals(arr1, arr2) {
```

Comment:
is this only for arrays, or generic?
either name it accordingly or handle objects too.
(There you would need to sort the properties, because 2 javascript objects would be equal if their properties have the same values, even when the order is not the same.)

can't lodash do that for you?

-> new issue